neqo-http3/tests/httpconn.rs

```diff
@@ -111,6 +112,38 @@ fn connect_peers(hconn_c: &mut Http3Client, hconn_s: &mut Http3Server) -> Option
     out.dgram()
 }
 
+fn connect_peers_with_rtt(
+    hconn_c: &mut Http3Client,
+    hconn_s: &mut Http3Server,
+    net_delay: u64,
+) -> (Option<Datagram>, Instant) {
+    let net_delay = Duration::from_millis(net_delay);
+    assert_eq!(hconn_c.state(), Http3State::Initializing);
+    let mut now = now();
+    let out = hconn_c.process(None, now); // Initial
+    now += net_delay;
+    let out = hconn_s.process(out.dgram(), now); // Initial + Handshake
+    now += net_delay;
+    let out = hconn_c.process(out.dgram(), now); // ACK
+    now += net_delay;
+    mem::drop(hconn_s.process(out.dgram(), now)); //consume ACK
```

Comment:
Rather than mem::drop() here, you should probably just use process_input() at the client, so that it doesn't generate the ACK and the server doesn't need to process it.  Then we don't drop packets in the test. Dropping stuff can affect other things, which might not alter this particular test, but it's probably worthwhile. You'll need to unwrap the datagram to do that, but I think that it's better.

You can also remove some of the extra delays then.

Reply:
We do not need to drop this packet because it is not a datagram. I will add an assertion.